manifests/config.pp

```diff
@@ -9,6 +9,11 @@
           unless  => "test /etc/alternatives/java -ef '${java::use_java_alternative_path}'",
         }
       }
+      if $java::use_java_home != undef {
+        file { '/etc/environment':
```

Comment:
This file is used for environment variables of all processes, so should not be directly managed by this module as any other lines would be erased or it would cause puppet resource conflicts.

Is there any other possible way to solve this? Or do we just have to leave JAVA_HOME settings up to the user to implement?